tests/test_core.py

```diff
@@ -619,6 +620,22 @@ def mock_service(call):
 
         self.services.register("Test_Domain", "TEST_SERVICE", mock_service)
 
+        self.calls_register = []
+        self.calls_remove = []
+
+        @ha.callback
+        def mock_event_register(event):
+            """Mock register event."""
+            self.calls_register.append(event)
+
+        @ha.callback
+        def mock_event_remove(event):
```

Comment:
No reason to register this for every test if you are only going to use it in 1 test.